components/Video/VideoCard/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { PropsWithChildren } from 'react'
+import Image from 'next/image'
+import Link from 'next/link'
+
+export interface VideoCardProps {
+  title: string
+  youtubeVideoID: string
+}
+
+const VideoCard = ({
+  title,
+  youtubeVideoID,
+  ...props
+}: PropsWithChildren<VideoCardProps>) => {
+  const videoURL = 'https://www.youtube.com/watch?v=' + youtubeVideoID
+  //const videoThumbnail = "https://img.youtube.com/vi/" + youtubeVideoID + "/maxresdefault.jpg";
+  const videoThumbnail =
```

Comment:
Might be easier to use template literals https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals